server/controllers/stock/reports/stock/inventory_report.js

```diff
@@ -0,0 +1,59 @@
+const { _, db, ReportManager, Stock, STOCK_INVENTORY_REPORT_TEMPLATE } = require('../common');
+
+/**
+ * @method stockInventoryReport
+ *
+ * @description
+ * This method builds the stock inventory report as either a JSON, PDF, or HTML
+ * file to be sent to the client.
+ *
+ * GET /reports/stock/inventory
+ */
+function stockInventoryReport(req, res, next) {
+  const data = {};
+  let options;
+  let report;
+
+  const optionReport = _.extend(req.query, {
+    filename : 'TREE.STOCK_INVENTORY_REPORT',
+    orientation : 'landscape',
```

Comment:
It looks like almost every report uses the following object:
```js
{
  orientation : 'landscape',
  footerRight : '[page] / [toPage]',
  footerFontSize : '8',
}
```

Could you put this object in `common.js`?  Then it will be available to use with `_.extend()`, [which can take multiple objects](https://lodash.com/docs/4.17.4#assignIn).  It would look something like this:
```js
const optionReports = _.extend(req.query, pdfOptions, {
  filename : 'TREE.STOCK_INVENTORY_REPORT',
});
```

Reply:
You are right -  I can see it in a lot of reports. So I have refactored it out like you say and put it in common.js :)
974cc6bfd44e4e4f85a98653ce301ca6bd187a80